src/index.css

```diff
@@ -373,7 +373,7 @@ main {
 .earn-indicator .activity-indicator {
   --size: 6px;
   position: absolute;
-  right: calc(var(--size) * -1);
+  right: calc(var(--size) * -1 - 3px);
```

Comment:
Sorry for being imprecise, I meant adding the margin only to the "Send" indicator. Since that has the "d" at the end which makes it look crammed. "Earn" and "Jam" are fine.

Reply:
I know what you mean, but this is depends on the language.. should we really do that?